packages/vitest-pool-workers/vitest.config.mts

```diff
@@ -5,5 +5,7 @@ export default defineConfig({
 		globalSetup: ["./test/global-setup.ts"],
 		exclude: [...configDefaults.exclude, "**/*.worker.test.ts"],
 		testTimeout: 30_000,
+		poolOptions: { forks: { singleFork: true } },
+		retry: 2,
```

Comment:
Was this intentional? This is in `vitest-pool-workers`.

Reply:
Probably not. I was having a lot of pain getting all the tests to run locally, so I probably added this in an attempt to resolve that.
But it is a bit out of context in this PR. So I'll remove. If it is actually helpful we can add it back in via another PR.